examples/data-literate/components/Layout.js

```diff
@@ -11,7 +11,9 @@ export default function Layout({ children, title = 'Home' }) {
         <meta name="viewport" content="initial-scale=1.0, width=device-width" />
       </Head>
       <div className="mx-auto p-6">
-        {children}
+        <div className="prose mx-auto">
```

Comment:
I don't think we want the prose in the Layout component - we want that only in DataLiterate. That way we can use the Layout component for non-markdown pages if needed.

Reply:
`prose` class could move to wrapper class or `main` in DataLiterate.

Reply:
Alright, will update it now